lib/main_common.pm

```diff
@@ -44,9 +45,9 @@ our @EXPORT = qw(
   any_desktop_is_applicable
   console_is_applicable
   boot_hdd_image
+  maybe_load_kernel_tests
```

Comment:
what the reason ? what ordering rules we have here ?

Reply:
I changed it by accident